app/_data/landing_page.yml

```diff
@@ -3,98 +3,106 @@ features:
     items:
```

Comment:
@fabianrbz I think the "Key features" heading text is kind of small in comparison to the heading text on a tile ("Service Hub") for example. I think maybe the "Key features" or section title header text should be larger than the heading of a tile.